packages/create-project/src/helpers/git.ts

```
@@ -0,0 +1,48 @@
+/* eslint-disable no-empty */
+import { execSync } from 'child_process';
+import path from 'path';
+import rimraf from 'rimraf';
+
+function isInGitRepository(): boolean {
+  try {
+    execSync('git rev-parse --is-inside-work-tree', { stdio: 'ignore' });
+    return true;
+  } catch (_) {}
```

This could hide potential unexpected errors